core/arch/arm/kernel/generic_boot.c

```diff
@@ -821,6 +825,29 @@ static void reset_dt_references(void)
 {
 }
 
+static void carveout_memory_range(void)
+{
+	const struct core_mmu_phys_mem *mem;
+	size_t nelems = 0;
+
+	for (mem = &__start_phys_ddr_overall_section;
+	     mem < &__end_phys_ddr_overall_section; mem++) {
+		nelems++;
+	}
```

Comment:
minor: remove braces, or `nelems = (__end_phys_ddr_overall_section - __start_phys_ddr_overall_section) / sizeof(*mem);`

Reply:
OK, I will use the second method.